readinessProbe/readiness/server.go

```diff
@@ -0,0 +1,59 @@
+package readiness
+
+import (
+	"context"
+	"io"
+	"net/http"
+
+	v1 "github.com/openshift/ocs-osd-deployer/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	listenAddr          string = ":8081"
+	readinessPath       string = "/readyz/"
+	NamespaceEnvVarName string = "NAMESPACE"
+)
+
+func Logic(client client.Client, managedOCSResource types.NamespacedName, w io.Writer) (bool, error) {
```

Comment:
[Suggestion] Have a better descriptive name, also make is private

```suggestion
func IsReady(client client.Client, managedOCSResource types.NamespacedName, w io.Writer) (bool, error) {
```